policyengine_us/parameters/gov/ed/pell_grant/efc/automatic_zero.yaml

```diff
@@ -1,9 +1,30 @@
 description: The Department of Education sets the Expected Family Contribution to zero if a family has income below this value.
 values:
-  2023-01-01: 27_000
+  2009-08-01: 30_000
```

Comment:
We're inconsistent about this but given the number of references, could you adopt this style to pair each value with the reference(s)?

https://github.com/PolicyEngine/policyengine-us/blob/6bdf9bb89ac989473eb80795ed2557b6f1206885/policyengine_us/parameters/gov/usda/snap/income/deductions/excess_shelter_expense/homeless/available.yaml#L20-L25

Reply:
Do you want me to do this for the marginal rate as well? In that one, each reference would be in multiple places.

Reply:
Do you want me to do this for the marginal rate as well? In that one, each reference would be in multiple places.

Reply:
Will I have to use `parameter(period).gov.ed.pell_grant.efc.automatic_zero.value`

Reply:
yes please

Reply:
ah shoot i see what you mean. applying this on scale parameters will create significant reference duplication.

@nikhilwoodruff what would you suggest here? we have many years of policy with changing values each year that we want to tie to specific references. should we make a separate scale for each year so we can attach the reference once for the full scale?

Reply:
Sorry for my delay- I think it's OK to add the reference at the highest level possible (such that all nodes below are from the reference). Filed https://github.com/PolicyEngine/policyengine-core/issues/115 which would then smooth out the parameter reference data.